### PR TITLE
docs: packages and apps

### PR DESCRIPTION
**Pull request by 0xkenj1**
# 🤖 Linear

Closes GIT-118 GIT-119 GIT-120

## Description
Improve docs across the entire the repo

## Checklist before requesting a review

-   [x] I have conducted a self-review of my code.
-   [x] I have conducted a QA.
-   [x] If it is a core feature, I have included comprehensive tests.

### CODE REVIEW

File: packages/data-flow/README.md [hunk 1]
```diff
@@ -43,3 +43,29 @@ const result = await eventsFetcher.fetcEventsByBlockNumberAndLogIndex(
...
+
+The `EventsProcessor` class is responsible for processing events in the processing pipeline.
+
+### [EventsFetcher](./src/eventsFetcher.ts)
+
+The `EventsFetcher` class is responsible for fetching events from the blockchain.
+
+### [StrategyRegistry](./src/strategyRegistry.ts)
+
+The `StrategyRegistry` stores strategy IDs in memory to populate strategy events with them.
```
Comment (0xnigir1) [inline packages/data-flow/README.md]:
```suggestion
The `StrategyRegistry` stores strategy IDs to populate strategy events with them given the Strategy address.
```

File: packages/data-flow/README.md [hunk 2]
```diff
@@ -43,3 +43,29 @@ const result = await eventsFetcher.fetcEventsByBlockNumberAndLogIndex(
...
+
+### [DataLoader](./src/data-loader/dataLoader.ts)
+
+The `DataLoader` is responsible for applying changesets to the database..
```
Comment (0xnigir1) [inline packages/data-flow/README.md]:
```suggestion
The `DataLoader` is responsible for applying changesets to the database.
```

File: packages/processors/README.md [hunk 3]
```diff
@@ -22,14 +22,55 @@ Available scripts that can be run using `pnpm`:
...
+const alloProcessor = new AlloProcessor(chainId, dependencies);
+const changeset = await this.alloProcessor.process(event);
+```
 
 ## API
 
-Describe your package's API here.
+### Processors
+
+This package provides the following repositories:
```
Comment (0xnigir1) [inline packages/processors/README.md]:
```suggestion
This package provides the following events' processors:
```

**Review by 0xnigir1 — DISMISSED** [review]
byutiful, just super minor suggestions

**Review by 0xnigir1 — APPROVED** [review]

**Review by jahabeebs — APPROVED** [review]
nice!

**Pull request merged**